libs/common.py

```diff
@@ -99,3 +101,17 @@ def _create_path(my_path):
         if e.errno != errno.EEXIST:
             logging.error(e)
             exit(1)
+
+def _generate_cluster_name_seed(cluster_name_seed):
+    _cluster_name_seed = cluster_name_seed
+    allowed_chars = string.ascii_lowercase + string.digits
+    for l in _cluster_name_seed:
+        if l not in allowed_chars:
+            logging.error('Cluster name seed is not valid: %s' % _cluster_name_seed)
```

Comment:
Can we also output a line about what is acceptable seed options? Something like
"Cluster name seed must contain only lowercase letters and digits"

Reply:
Done